sim/simflash/src/lib.rs

```diff
@@ -39,6 +39,7 @@ fn ebounds<T: AsRef<str>>(message: T) -> ErrorKind {
     ErrorKind::OutOfBounds(message.as_ref().to_owned())
 }
 
+#[allow(dead_code)]
```

Comment:
We could also just remove the function, although it would then be work to figure out how to add it back in if we did want to return errors.